### ISSUE TITLE
Truncation psi not doing anything?

### ISSUE THREAD

**Issue by mewo2** [lucidrains/lightweight-gan#51]
This may be me missing something screamingly obvious, but I can't see anywhere in the code where the `trunc_psi` parameter is actually used. In particular, `generate_truncated` looks like this:

```python
    def generate_truncated(self, G, style, trunc_psi = 0.75, num_image_tiles = 8):
        generated_images = evaluate_in_chunks(self.batch_size, G, style)
        return generated_images.clamp_(0., 1.)
```

That doesn't seem to do the truncation at all, and experimentation backs this up: varying psi has no discernible effect on the generated images.

**Comment by lucidrains** [comment]
@mewo2 yup, i tried to add truncation but wasn't getting good results. i've remove all traces of it in the latest commit!
